### PR TITLE
pass obt verify on v6

### PR DESCRIPTION
**Pull request by JakeChampion**
We switched from scss-lint to sass-lint

### CODE REVIEW

File: src/stylesheets/livefyre_widget_overrides.scss [hunk 1]
```diff
@@ -469,9 +469,9 @@
 		}
 
 		.fyre-comment-flag-mobile-btn {
-			
+
```
Comment (onishiweb) [inline src/stylesheets/livefyre_widget_overrides.scss]: whitespace additions?

Comment (JakeChampion) [inline src/stylesheets/livefyre_widget_overrides.scss]: whitespace removal :-)

Comment (onishiweb) [inline src/stylesheets/livefyre_widget_overrides.scss]: Ah ok, sorry was commenting on the newline here, should probably not be a line here no?

Comment (JakeChampion) [inline src/stylesheets/livefyre_widget_overrides.scss]: I don't think it is a new line, just Github getting confused. Try viewing without whitespace changes -- https://github.com/Financial-Times/o-comments/pull/11/files?w=1

Comment (onishiweb) [inline src/stylesheets/livefyre_widget_overrides.scss]: cool 👍

**Pull request merged**